readme.md

```diff
@@ -34,20 +34,20 @@ import 'package:grinder/grinder.dart';
 main(args) => grind(args);
 
 @Task('Initialize stuff.')
-init(GrinderContext context) {
-  context.log("Initializing stuff...");
+init() {
+  log("Initializing stuff...");
 }
 
 @Task('Compile stuff.')
 @Depends(init)
-compile(GrinderContext context) {
-  context.log("Compiling stuff...");
+compile() {
+  log("Compiling stuff...");
 }
 
 @DefaultTask('Deploy stuff.')
```

Comment:
OK, makes sense. It struck me as odd - when scanning that script - that not all the tasks started with `@Task...`. But I see your reasoning.